src/layer.yaml

```diff
@@ -1,24 +1,12 @@
-"includes":
-- "layer:options"
-- "layer:basic"
-- "interface:tls-certificates"
-- "layer:openstack"
-- "interface:cinder-backend"
-"exclude": [".travis.yml", "tests", "tox.ini", "test-requirements.txt", "unit_tests",
-  ".gitignore", ".gitreview", ".zuul.yaml", "README.md", "requirements.txt"]
-"options":
-  "basic":
-    "use_venv": !!bool "true"
```

Comment:
I think these options are for `layer:basic`. Are you really really sure this is the right move to remove this option for example? Removing this option may prevent the charm from installing properly on some Ubuntu series if I remember correctly.